### ISSUE TITLE
Record fields missing secondaryFiles

### ISSUE THREAD

**Issue by sivkovic** [common-workflow-language/common-workflow-language#490]
Currently it is impossible to set secondaryFiles for CommandInputRecordField. If I want to have the record which looks like:

```
{
  "name": "RecordName",
  "type": "record",
  "fields": [{
    "name": "file",
    "type": "File",
  },
  {
    "name": "paired_end",
    "type": "int"
  }]
}
```

I wouldn't be able to specify secondaryFiles which goes with file. So basically if I need any information which I need in addition to my file value, like paired-end or sample-id, which I want to use for grouping files in some way, I'm loosing ability to use secondaryFiles feature.

**Comment by tetron** [comment]
You're right, this has come up before (although I can't find an earlier issue).  We need to fix this.

**Comment by tetron** [comment]
Will be fixed in #636
